ocf_datapipes/transform/numpy/save_t0_time.py

```diff
@@ -0,0 +1,27 @@
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+from ocf_datapipes.utils.consts import BatchKey, NumpyBatch
+
+
+@functional_datapipe("save_t0_time")
+class SaveT0TimeIterDataPipe(IterDataPipe):
```

Comment:
Save? or is it get? Save to me implies we save it to a file?

Reply:
Ah, this is names this as its just directly one of the np_batch_procesors used in PP, it saves the t0 time into the batch